tf2x.yml

```diff
@@ -0,0 +1,15 @@
+name: tf2.10X
+channels:
+  - defaults
+  - conda-forge
+  - anaconda
+dependencies:
+  - python=3.9
+  - cudnn
+  - nibabel
+  - gputil
+  - ants
+  - pip
+  - pip:
+    - tensorflow==2.10.0
+    - git+https://github.com/pnlbwh/conversion.git
```

Comment:
Overall, I do not understand how this recipe supersedes the old recipe as you have omitted many packages. Please explain @RyanZurrin

Reply:
The omitted packages were either installed as dependencies for other packages or were not needed to run the CNN-Masking. But please verify this of course.

Reply:
> But please verify this of course.

Ryan, this requires trial and error. You need to verify it. At least `scikit-image==0.16.2` is required for `-filter scipy` flag.

Reply:
Or a `>=` version.

Reply:
okay, let me add this to the recipe, and then retest things. I will also change the name to a more human-readable name. As for the ANTs, it is part of the conda-forge channel.

Reply:
if you activate the pnlpipe3-cuda10 env and check where the ANTs are from, you will see it is from the conda-forge and was not installed from the pnlbwh channel.